docs/pipelines/add-pipeline.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Add a Pipeline
+intro: By adding a pipeline to your environment's resources, you can automate their deployment by using customized pipeline steps.
+links:
+    overview:
+    quickstart:
+    previous:
```

Comment:
```suggestion
    previous:
        - /docs/applications/add-application
```

I think this post could be connected with the introduction guide (`previous` and `next` links); keeping in mind to update the "Add Pipeline Step" previous link to this post.

What do you think @plantaz?

Reply:
I'm not sure if this should be included in the guide because the default pipeline, which is automatically created when an application is created, may be sufficient for some applications.

Reply:
I'm not sure if this should be included in the guide or if it's more suitable as a related topic. 

The default pipeline, which is automatically created when an application is created, may be sufficient for some applications, similar to how configuration files may or may not be required for a successful deployment.

What do you think about it?

Reply:
I agree, and it can be improved upon on another PR